src/bridge/command.rs

```diff
@@ -180,20 +214,40 @@ fn platform_which(bin: &str) -> Option<String> {
     None
 }
 
+#[cfg(target_os = "macos")]
+pub fn is_tty() -> bool {
+    !std::io::stdout().is_terminal()
+}
+
 #[cfg(target_os = "macos")]
 fn nvim_cmd_impl(bin: String, mut args: Vec<String>) -> TokioCommand {
```

Comment:
It seems like we should be able to share the implementation between this and `create_platform_shell_command`

The only difference seems to be that one of the functions constructs a `TokioCommand` and the other one a `std::Command`. So perhaps we should create a function that builds just the arguments?

Reply:
done